maro/simulator/scenarios/vm_scheduling/business_engine.py

```diff
@@ -635,6 +675,7 @@ def _overload(self, pm_id: int):
 
         if self._kill_all_vms_if_overload:
             for vm_id in vm_ids:
+                self._total_incomes -= self._live_vms[vm_id].get_income_till_now(tick)
```

Comment:
since overload is called before be._update_income; and the vm.get_income_till_now will take the income of current tick into account. one more unit_price is decreased?

Reply:
How do you solve it? You didn't reply me.